src/main/java/org/influxdb/dto/utils/CheckTags.java

```diff
@@ -0,0 +1,45 @@
+package org.influxdb.dto.utils;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * {Purpose of This Type}.
+ *
+ * {Other Notes Relating to This Type (Optional)}
+ *
+ * @author Brenton Poke
+ *
+ */
+public final class CheckTags {
+  static final String NAMEREGEX = "([a-zA-Z0-9-_]+)";
```

Comment:
Can you please point here to the official influxdb documentation where these restrictions are described and which influxdb versions are affected.

Reply:
There are no restrictions described in the documentation, which is the reason newline characters weren't being handled. I asked in #684 what other restrictions there should be, no feedback. These were reasonable starting points for that discussion.

Reply:
OK, but i don't want to add restrictions here which are guessed, this will lead to a plethora of bug reports points not able to be written because of that.

Reply:
Then I will only restrict newline and carriage return for #684, and maybe a new issue can be created if problems arise.